src/main/java/cart/service/MemberService.java

```diff
@@ -0,0 +1,67 @@
+package cart.service;
+
+import cart.controller.dto.MemberDto;
+import cart.domain.Member;
+import cart.domain.MemberPassword;
+import cart.domain.MemberRole;
+import cart.exception.ErrorCode;
+import cart.exception.GlobalException;
+import cart.persistence.dao.MemberDao;
+import cart.persistence.entity.MemberEntity;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+public class MemberService {
+
+    private final MemberDao memberDao;
+
+    public MemberService(final MemberDao memberDao) {
+        this.memberDao = memberDao;
+    }
+
+    public long save(final MemberDto memberDto) {
```

Comment:
```suggestion
    @Transactional
    public long save(final MemberDto memberDto) {
```

`@Transactional(readOnly=true)` 빠르게 잘 학습하셨네요 👍